### PR TITLE
URLhaus feed

### PR DESCRIPTION
**Pull request by gal-forer**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://github.com/demisto/etc/issues/45440)

## Description
Urlhaus fed integration.

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/FeedURLhaus/Integrations/FeedURLhaus/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## URLhaus
```
Comment (merit-maita) [inline Packs/FeedURLhaus/Integrations/FeedURLhaus/README.md:1]: for the reademe, revert to the previous version of the file, you have in the first pr.

**Review by merit-maita — DISMISSED** [review]
don't you have a customer indicator's field ?

File: Packs/FeedURLhaus/Integrations/FeedURLhaus/README.md (hunk 1 above)
Comment (gal-forer) [reply Packs/FeedURLhaus/Integrations/FeedURLhaus/README.md:1]: I dont understand it is the same as it was in the last pr, do I need to add something ? @merit-maita

**Comment by merit-maita** [comment]
> don't you have a customer indicator's field ?

what about it?

**Comment by bakatzir** [comment]
Hey, I would love to join the demo for this.

File: Packs/FeedURLhaus/Integrations/FeedURLhaus/FeedURLhaus.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                    'Tags': 'tags',
+                    'Download URL': 'urlhaus_link',
+                    'Reported By': 'reporter'
+                }
+            }
+            chosen_urls.append(base_url + suffix)
+    params["indicator_type"] = FeedIndicatorType.URL
+    params['ignore_regex'] = '#'
+    params['url'] = chosen_urls
+    feed_main('URLhaus Feed', params, 'urlhaus-feed-')
```
Comment (yaakovi) [inline Packs/FeedURLhaus/Integrations/FeedURLhaus/FeedURLhaus.py:37]:
```suggestion
    feed_main('URLhaus Feed', params, 'urlhaus-')
```

File: Packs/FeedURLhaus/Integrations/FeedURLhaus/FeedURLhaus.yml [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+  - never
+  - interval
+  - indicatorType
+  - suddenDeath
+  required: false
+  type: 17
+- display: Trust any certificate (not secure)
+  name: insecure
+  type: 8
+  required: false
```
Comment (yaakovi) [inline Packs/FeedURLhaus/Integrations/FeedURLhaus/FeedURLhaus.yml:52]: Move to the end.
Why isn't there a `proxy` parameter?

**Comment by tschanfeld** [comment]
@gal-forer @yaakovi Is the readme file supposed to be empty? I reviewed the description md and the yml file.

**Comment by gal-forer** [comment]
> @gal-forer @yaakovi Is the readme file supposed to be empty? I reviewed the description md and the yml file.

I've added a readme @tschanfeld

**Comment by tschanfeld** [comment]
@gal-forer Why would all the parameters in the readme be listed as required? While I don't know about all of them, I think trust any certificate and use system proxy settings are optional.

**Review by yaakovi — APPROVED** [review]
why did you remove the test playbook?

**Comment by gal-forer** [comment]
> why did you remove the test playbook?

I thought you wanted me to @yaakovi 
image

**Pull request merged**